app/models/template.rb

```diff
@@ -31,10 +31,6 @@ def self.template_includes
     []
   end
 
-  def preview_host_collection
```

Comment:
Thanks for the hint, it was definitely not the aim of this PR to break remote execution ;-)